tests/core/apt/task.yaml

```diff
@@ -2,6 +2,8 @@ summary: Ensure that the core systems have a special apt placeholder
 
 systems: [ubuntu-core-*]
 
+tags: [ basic ]
```

Comment:
this seems a test that is enough to run with PRs, it involves something that shouldn't change often at all and isn't really related to hardware or kernel

Reply:
sure, removed